example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml

```diff
@@ -4486,6 +4486,128 @@ spec:
                 baseImage:
                   description: Thanos base image if other than default.
                   type: string
+                grpcServerTlsConfig:
+                  description: 'GRPCServerTLSConfig configures the GRPC server from
```

Comment:
I think it's called `gRPC`? 
![image](https://user-images.githubusercontent.com/6950331/76337417-3b4f9b00-62ef-11ea-81bf-291188c68366.png)

Reply:
Fixed